codalab/lib/bundle_cli.py

```diff
@@ -1054,6 +1054,21 @@ def do_store_command(self, args):
                 "url": args.url,
                 "authentication": args.authentication,
             }
+            if args.url is not None:
```

Comment:
Rather than hardcoding azfs://, azure_blob, etc., can we instead just enumerate through the constants defined here (https://github.com/codalab/codalab-worksheets/blob/545106e1bf9599f608597d9dcbe82391788d6dab/codalab/common.py#L171-L190)?

This will reduce code duplication.

Reply:
Thanks Ashwin, already changed to use common classes.

Reply:
Can we simplify this further? We can just loop through the values of `StorageURLScheme` and `StorageType` -- or we might need to define a dict in common.py mapping `StorageURLScheme` to `StorageType`

Reply:
Cool! I will define a dict in common.py to make it more clear, in case there are some new type of storage

Reply:
Feel feel to resolve a conversation when we're done with it